Dangerfile

```diff
@@ -98,17 +83,19 @@ def checkForEnginesVersion(file)
   warn("`.nvmrc` was modified. Remember to update node version on `.travis.yml`, `package.json` and `README`!") if file && file == ".nvmrc"
 end
 
-def checkForWeaklyTypedFunctionReturn(line)
+def checkForWeaklyTypedFunctionReturn(file)
   # Warn when a TypeScript file has a new function returning <any> instead of strongly typed.
   # There are several situation that need to return just 'any', so to avoid having too many false positives
   #   we are checking just <any> for now
-  warn("Possibly returning 'any' in a function, prefer having a strongly typed return. `#{file}` at line `#{line}`.") if line =~ /<any>/im
+  returnAnyMatches = checkForRegex(file, /\+.*<any>/im)
+  returnAnyMatches.each do |returnAnyMatch|
+    warn("Possibly returning 'any' in a function, prefer having a strongly typed return. `#{file}` at line `#{returnAnyMatch}`.") if returnAnyMatch
+  end
 end
 
 def checkForNpmInstallGlobal(file)
   # Warn developers that they are not supposed to use this flag
-  fileDiff = git.diff_for_file(file)
-  npmInstallMatches = fileDiff.patch.scan(/\+.*npm install (-g|--global)/)
+  npmInstallMatches = checkForRegex(file, /\+.*npm install (-g|--global)/)
```

Comment:
can we ignore certain files?

some PRs are _failing_ due to `npm install -g yarn` instruction on `README`, ex: https://github.com/indigotech/br-pro120-jogo_das_forcas-server-api/pull/10#issuecomment-287921670